bazaar/Cargo.toml

```diff
@@ -0,0 +1,93 @@
+[package]
+name = "pallet-encointer-bazaar"
+version = "0.3.6"
+authors = ["encointer.org <alain@encointer.org>"]
+edition = "2018"
+
+[dependencies]
+
+[dependencies.rstd]
+default-features = false
+package = "sp-std"
+version = "2.0.0"
+
+[dependencies.codec]
+default-features = false
+package = "parity-scale-codec"
+features = ["derive"]
+version = "1.3.0"
+
+[dependencies.encointer-currencies]
+default-features = false
+path = "../currencies"
+package = "pallet-encointer-currencies"
+
+[dependencies.runtime-io]
+default-features = false
+package = "sp-io"
+version = "2.0.0"
```

Comment:
master has progressed to 

```
git = "https://github.com/paritytech/substrate.git" 
rev = "a208da16"
```
see other pallets.

please update your dependencies too